internal/analytics/segment_reporter.go

```diff
@@ -17,6 +17,9 @@ import (
 const (
 	kubeSystemNSName  = "kube-system"
 	unknownIdentityID = "00000000-0000-0000-0000-000000000000"
+
+	typedCommand  = "typed"
+	buttonCommand = "buttonClick"
```

Comment:
Please create enum (even if you don't agree about passing enum to the ReportCommand function, in such case just make it unexported):

```suggestion
type CommandTrigger string

const (
	ButtonClickCommandTrigger  = "buttonClick"
	TypedMessageCommandTrigger = "typedMessage"
)
```

Reply:
done, (unexported)